google-cloud-asset/lib/google/cloud/asset/v1/asset_service_client.rb

```diff
@@ -371,30 +413,231 @@ def export_assets \
           #
           #   # TODO: Initialize `parent`:
           #   parent = ''
-          #
-          #   # TODO: Initialize `content_type`:
-          #   content_type = :CONTENT_TYPE_UNSPECIFIED
-          #
-          #   # TODO: Initialize `read_time_window`:
-          #   read_time_window = {}
-          #   response = asset_client.batch_get_assets_history(parent, content_type, read_time_window)
+          #   response = asset_client.batch_get_assets_history(parent)
 
           def batch_get_assets_history \
               parent,
-              content_type,
-              read_time_window,
               asset_names: nil,
+              content_type: nil,
+              read_time_window: nil,
```

Comment:
Breaking changes: Positional params changed to keyword args.

Reply:
@dazuma ^

Reply:
I'll check upstream on this one.